charts/k8ssandra-cluster/templates/stargate/stargate.yaml

```diff
@@ -0,0 +1,109 @@
+{{- if .Values.stargate.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-k8ssandra-stargate
+  labels:
+    app: {{ .Release.Name }}-k8ssandra-stargate
+{{ include "k8ssandra-cluster.labels" . | indent 4 }}
+spec:
+  replicas: {{ default 1 .Values.stargate.replicas }}
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-k8ssandra-stargate
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}-k8ssandra-stargate
+    spec:
+      initContainers:
+          - name: wait-for-cassandra
+            image: alpine:3.12.2
+            imagePullPolicy: IfNotPresent
+            command:
+              - sh
+            args:
+              - -c
+              - |
+                echo "Waiting for at least one cassandra node to finish bootstrapping..."
+                while ! nslookup {{ .Values.clusterName }}-seed-service.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain | default "cluster.local" }}; do
+                    sleep 5
+                done
+                echo "Cassandra is ready. Starting Stargate..."
+      containers:
+        - name: {{ .Release.Name }}-k8ssandra-stargate
+  {{- if eq .Values.stargate.containerImage "auto" }}
```

Comment:
Do you think that `containerImage` and `clusterVersion` should be determined by what version of Cassandra is used? I tend to think so. We can address that in a follow up PR.

Reply:
Yes -- but since we're hardcoding that in our cass-operator config, I think there's no way to do it?